src/lib.rs

```diff
@@ -9,24 +9,22 @@
 #![cfg_attr(feature = "unstable", feature(core_intrinsics))]
 #![cfg_attr(feature = "unstable", feature(const_generics))]
```

Comment:
Is this still needed now that const generics w/ `usize` is stable?

Reply:
It isn't, fixed in 0699e4e, thanks!